module-alias.json

```diff
@@ -0,0 +1,4 @@
+{
+    "yahoosspBidAdapter": "yahooAdsBidAdapter",
```

Comment:
This made for backward compatibility for out IBP logic
IBP would support prebid 8.x and 9.x versions with those module renaming